NanoVNASaver/NanoVNASaver.py

```diff
@@ -609,11 +609,16 @@ def serialButtonClick(self):
     def startSerial(self):
         with self.serialLock:
             self.serialPort = self.serialPortInput.currentData()
+            logger.debug("current data %s", self.serialPortInput.currentData())
+            logger.debug("current text %s", self.serialPortInput.currentText())
             if self.serialPort == "":
                 self.serialPort = self.serialPortInput.currentText()
             logger.info("Opening serial port %s", self.serialPort)
             try:
-                self.serial = serial.Serial(port=self.serialPort, baudrate=115200)
+                if "MR100" in self.serialPortInput.currentText():
+                    self.serial = serial.Serial(port=self.serialPort, baudrate=57600)
```

Comment:
Really a buadrate or just set to identify mr100 hardware in the detection part?

Reply:
my mr100 needs 57600, not working on 115200.
So I set wen detected in name and use it to detect when I have only serial.